MatrixTest.java

```diff
@@ -0,0 +1,163 @@
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import org.junit.Test;
+
+
+public class MatrixTest{
+    //Matrix(double[][] A){}
+
+    //Matrix(double[][] A, int m, int n){}
+    @Test
+    public void baseCtorSameLength(){
```

Comment:
AH... I guess that does make sense. Alright, I'm fine with it. Just didn't get the abbreviation at first. Thanks @ccampell for explaining.